src/Api/ApiClientBuilder.php

```diff
@@ -93,6 +93,7 @@ public function build()
         }
 
         $guzzleClient->setDefaultOption('exceptions', $this->httpExceptions);
+        $guzzleClient->setDefaultOption('verify',false);
```

Comment:
You can add a method in the `ApiClientBuilder` that set's whether the verify option is true or false.

Then in the `Athena.php` file, where the `ApiClientBuilder` is called, you pass the pass the value to the builder using the settings class.

Example:

`https://github.com/athena-oss/plugin-php/blob/master/src/Athena.php#L278`